src/commands/database-remove.js

```diff
@@ -3,10 +3,8 @@
 var Command = require("../command");
 var requireInstance = require("../requireInstance");
 var requirePermissions = require("../requirePermissions");
-var request = require("request");
+var DatabaseRemove = require("../database/remove");
```

Comment:
@bkendall 
I convert new classes to Typescript, but not sure how to import them from a javascript file.

Reply:
Since it's a default export, you want:

```suggestion
var DatabaseRemove = require("../database/remove").default;
```